sql/migrate/migrate.go

```diff
@@ -75,7 +75,12 @@ type (
 	// PlanApplier wraps the methods for planning and applying changes
 	// on the database.
 	PlanApplier interface {
+		// PlanChanges returns a migration plan for applying the given changeset.
 		PlanChanges(context.Context, string, []schema.Change) (*Plan, error)
+
+		// ApplyChanges responsible for applying the given changeset.
+		// An error may return from ApplyChanges if the driver does not
+		// know how to execute a change.
```

Comment:
i think less "personification" of components is better :-)
```suggestion
		// An error may return from ApplyChanges if the driver is
		// unable to execute a change.
```